muon/domain/messages.js

```diff
@@ -12,19 +12,17 @@ var schema = Joi.object().keys({
    created: Joi.date().timestamp('javascript'),
    target_service: Joi.string().min(3).required(),
    origin_service: Joi.string().min(3).required(),
-   url: Joi.string().uri().required(),
```

Comment:
url is a protocol concern. I introduced it properly into rpc in request.headers.url instead of here.